docs/src/Checkbox.doc.js

```diff
@@ -85,64 +93,35 @@ card(
     name="Example: Accessibility"
```

Comment:
@AlbertCarreras Please revisit the `description` of all examples. We should remove `If you use padding instead of margin around your labels, the clickable area will be larger` in this case

Reply:
DONE